src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/validators.py

```diff
@@ -289,6 +290,33 @@ def _validate_other_parameters(namespace, formula=None):
             namespace.os_type = formula.os_type
 
 
+def _validate_artifacts(namespace):
```

Comment:
Maybe I'm late to the party. But the method name is misleading. It modifies the artifacts properties of namespace. The name `validate` indicate a read-only operations.